bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java

```diff
@@ -548,17 +560,31 @@ public void remove(byte[] key) throws IOException {
         @Override
         public void clear() {
             writeBatch.clear();
+            batchCount = 0;
```

Comment:
If the user invoke flush() directly, we should also reset the batchCount

Reply:
I don't think so. Users can flush multiple times using the same batch. Even you flush the batch, the batch's content doesn't clean up. If we reset it after flushing, the count is not accurate. So I follow the API to reset the batch.

Reply:
If manual flush is completed and the batchCount reaches the batchSize again, does it mean that the operations in this batch will be applied to the database twice?